cartridge/lua-api/get-topology.lua

```diff
@@ -200,12 +114,80 @@ local function get_topology()
     end
 
     local active_leaders = failover.get_active_leaders()
+    local refined_uri = topology.refine_servers_uri(topology_cfg)
 
+    --- Instance general information.
+    -- @tfield string alias
+    --   Human-readable instance name.
+    -- @tfield string uri
+    -- @tfield string uuid
+    -- @tfield boolean disabled
+    -- @tfield string status
+    --   Instance health.
+    -- @tfield string message
+    --   Auxilary health status.
+    -- @tfield ReplicasetInfo replicaset
+    --   Circular reference to a replicaset.
+    -- @tfield number priority
+    --   Leadership priority for automatic failover.
+    -- @tfield number clock_delta
+    --   Difference between remote clock and the current one (in
+    --   seconds), obtained from the membership module (SWIM protocol).
+    --   Positive values mean remote clock are ahead of local, and vice
+    --   versa.
+    -- @table ServerInfo
     for _, instance_uuid, server in fun.filter(topology.not_expelled, topology_cfg.servers) do
-        local srv = get_server_info(members, instance_uuid, server.uri)
```

Comment:
To be honest I'd prefer to see this logic in separate function.

It looks a bit strange to describe arguments of function part right inside function - usually if you want to describe that some piece of code does you write extended comment and it's pure description for people (without ldoc).

Reply:
This function should be executed in a context (it modifies `members` table), that's why i merged them.

Reply:
Well, remove ldoc here. And rewrite comment for humans